plugins/pluginmanagers/lz-n.nix

```diff
@@ -209,6 +247,15 @@ lib.nixvim.plugins.mkNeovimPlugin {
     globals.lz_n = lib.modules.mkAliasAndWrapDefsWithPriority id options.plugins.lz-n.settings;
     plugins.lz-n.luaConfig.content = mkIf (cfg.plugins != [ ]) ''
       require('lz.n').load( ${toLuaObject cfg.plugins})
+
+      -- Set up lz.n keybinds {{{
+      do
+        local __lz_n_keymaps = ${toLuaObject (map removeDeprecatedMapAttrs cfg.keymaps)}
+        for _, map in ipairs(__lz_n_keymaps) do
+          require('lz.n').keymap(map.plugin).set(map.mode, map.key, map.action, map.options)
+        end
```

Comment:
Thoughts on serializing the whole table and iterating in lua **vs** iterating in nix and rendering multiple `"require('lz.n').keymap(${plugin}).set"` lines?

I suspect the latter may be easier to read in `nixvim-print-init` and was the approach we ended up taking in more recent code, e.g. `lsp.servers` :thinking:

E.g:
```nix
''
do
  ${lib.concatMapStringsSep "\n" (
    { plugin, mode, key, action, options, ... }:
    "require('lz.n').keymap(${toLuaObject plugin}).set(${toLuaObject mode}, ${toLuaObject key}, ${toLuaObject action}, ${toLuaObject options})"
  ) cfg.keymaps}
end
''
```

Reply:
I like it, changed the implementation